cmd/clusterctl/clusterdeployer/clusterclient/clusterclient.go

```diff
@@ -406,15 +408,32 @@ func newDeleteOptions() *metav1.DeleteOptions {
 }
 
 // TODO: Test this function
-func (c *client) UpdateClusterObjectEndpoint(controlPlaneIP, clusterName, namespace string) error {
+func (c *client) UpdateClusterObjectEndpoint(clusterEndpoint, clusterName, namespace string) error {
```

Comment:
Do we know what's using this method currently?

Reply:
clusterdeployer.go in `updateClusterEndpoint`